gae/gae.go

```diff
@@ -47,15 +47,20 @@ func NewMartini(option MartiniOption) *ExMartini {
 				continue
 			}
 
-			id := uuid.NewUUID()
+			id, err := uuid.NewRandom()
```

Comment:
Please call `uuid.New` to panic with error.
I prefer to panic rather than to ignore errors.